blitz_api/models.py

```diff
@@ -232,6 +245,42 @@ def offer_free_membership(self, membership):
             )
         self.save()
 
+    def has_membership_active(self):
+        today = timezone.now().date()
+        return \
+            self.membership and \
+            self.membership_end and \
+            self.membership_end >= today
+
+    def has_to_receive_first_notify(self):
```

Comment:
The name of the function is not really accurate since it's more `has_to_receive_notification` or `has_to_receive_new_notification`